### PR TITLE
Add detailed instructions for adding a site

### PR DESCRIPTION
**Pull request by jamiecobbett**
We need to make this process easy and so that anyone on 2nd line can complete
it, without arcane knowledge of Transition.

cc @bradleywright @jennyd @issyl0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,11 +5,87 @@ This repository holds two things:
...
+
+### Adding extra config to the site
+
+The generated file only contains fields which are absolutely required.
+
+Check that the `tna_timestamp` field is set. The rake task will try to find the latest archive by scraping the National Archives site, but this doesn't always work. You can find a list of the timestamps the National Archives have by going to their site, for example: [here are the timestamps for this Cabinet Office site](http://webarchive.nationalarchives.gov.uk/*/http://download.cabinetoffice.gov.uk>)
+
+Some of the optional fields are frequently used:
+
+* `aliases` - Almost every site will aliases to cover the host name with/without `www.` (depending on which host was used in the ``host`` field)
```
Comment (issyl0) [inline README.md]: "will have" / "will need" ?

File: README.md [hunk 2]
```diff
@@ -5,11 +5,87 @@ This repository holds two things:
...
+* `options` - used to specify when a site's URLs have significant querystring parameters.
```
Comment (issyl0) [inline README.md]: Nitpick: For consistency, could you remove the full stop here or add it to the end of other bulleted lines?

File: README.md (hunk 1 above)
Comment (jamiecobbett) [inline README.md]: Rebased.

File: README.md (hunk 2 above)
Comment (jamiecobbett) [inline README.md]: Sure - rebased.

File: README.md [hunk 3]
```diff
@@ -5,11 +5,87 @@ This repository holds two things:
...
+```
+  rake new_site[abbr,whitehall_slug,host]
+```
+
+``abbr`` should be of the form:
+```
+  <owning_organisation_abbreviation>_<abbreviated_site_name>
+```
+
+Example for the "SPIRE Export Control" site which is owned by the Department for Business, Innovation and Skills':
```
Comment (issyl0) [inline README.md]: There's a trailing ' before the colon.

**Comment by issyl0** [comment]
Apart from my punctuation nitpicking, it looks good. :sunny:

File: README.md (hunk 3 above)
Comment (jamiecobbett) [inline README.md]: Fixed.

**Comment by bradleywright** [comment]
:+1:

**Pull request merged**